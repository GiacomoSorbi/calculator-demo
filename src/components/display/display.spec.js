import * as React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Display from './'

describe('Display should', () => {
  it('exist', () => {
    expect(Display).toBeDefined()
  })

  it('have a consistent structure', () => {
    const wrapper = mount(<Display displayedText='(2+2)*(2+2)' />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
