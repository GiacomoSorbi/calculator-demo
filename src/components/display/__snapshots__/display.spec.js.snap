// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display should have a consistent structure 1`] = `
<Display
  displayedText="(2+2)*(2+2)"
>
  <div
    className="display-container"
  >
    <span>
      (2+2)*(2+2)
    </span>
  </div>
</Display>
`;
